PowerShell Contributing.md

```diff
@@ -0,0 +1,479 @@
+Why not give back? You may learn a thing or two on the way, and the PowerShell OSS community is very helpful and friendly too.
+
+If you would like a more thorough reasoning, I can recommend [Joel's blogpost on this](https://vexx32.github.io/2019/01/03/Contributing-Open-Source/).
+
+## Ten ways to contribute
+
+### 1. File a bug report
+
+> Requires
+> 
+> - GitHub account
+
+If you use PowerShell as your daily driver, or use it often on side projects, you may eventually run into something odd.
+
+File that bug right here: <https://github.com/PowerShell/PowerShell/issues>
+
+This is what you will be asked for when filing a bug report:
+
+- Make sure you are able to reproduce it on the [latest released version](https://github.com/PowerShell/PowerShell/releases)
+- Search the [existing issues](https://github.com/PowerShell/PowerShell/issues)
+- Refer to the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md)
+- Refer to the [known issues](https://docs.microsoft.com/powershell/scripting/whats-new/known-issues-ps6?view=powershell-7)
+- List steps to reproduce the bug
+- Explain what you expected to happen
+- Describe what actually happened instead
+- Paste in relevant environment info: output of `$PSVersionTable` and if any, the error message
+
+When in doubt, just share what you encountered to check if it's a bug in PowerShell or not.
+Good places to share are:
+
+- [Twitter](https://www.twitter.com), tag your tweet with #pshelp
+- The [PowerShell Discord channel](https://aka.ms/psdiscord)\*
+- The [PowerShell Slack channel](https://aka.ms/psslack)\*
+- The [PowerShell IRC channel](https://poshcode.org/#irc)\*
+- The [PowerShell/PowerShell Gitter channel](https://gitter.im/PowerShell/PowerShell)
+
+\*) Discord, Slack and IRC are bridged via a channel called #Bridge. Read more about this on <poshcode.org>.
+
+### 2. Help confirm bugs
+
+> Requires
+> 
+> - GitHub account
+
+Confirming bugs is going through recent bug report issues and trying to reproduce the bug.
+This helps assess if a reported bug is really a bug or not.
+
+A lot of new bug reports are going to have the ['Issue-Discussion'](https://github.com/PowerShell/PowerShell/issues?q=is%3Aissue+is%3Aopen+label%3AIssue-Discussion) label, instead of 'Issue-Bug'. 
+This is because when you first open an issue on the PowerShell project, the automatic label for a bug report is Issue-Discussion, which typically won't be changed until a maintainer or collaborator has seen that the bug has been confirmed / reproduced by others or can reproduce it themselves.
+
+You can comment in the issue whether or not you could repro it, and on which version of PowerShell.
+Always try to reproduce a bug on the latest version of PowerShell.
+Make sure you read the [FAQ](https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md) and the [known issues](https://docs.microsoft.com/powershell/scripting/whats-new/).
```

Comment:
Dead link. The one on line 52 does work.

Reply:
Thanks! Fixed in an upcoming commit.